src/components/navbar/Navbar.jsx

```diff
@@ -48,7 +47,7 @@ const Navbar = () => {
         </div>
 
         <NavDropdown
-          navLinks={mobileNavLinks}
+          navLinks={topNavLinks}
```

Comment:
I didn't see the need to two different constants because mobile nav and browser nav should contain the same links.